docs/user-guide/secrets/secret-volume-files-pod.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-files-test-pod
+spec:
+  containers:
+    - name: test-container
+      image: kubernetes/mounttest:0.1
+      command: [ "/mt", "--file_content=/etc/secret-volume/special/path/__special_thing.dat" ]
+      volumeMounts:
+          # name must match the volume name below
+          - name: secret-volume
+            mountPath: /etc/secret-volume
+  volumes:
+    - name: secret-volume
+      secret:
+        items:
+          - name: test-secret
```

Comment:
as per a comment in a different PR, this might want to change to:

```
volumes:
- name: secret-volume
  secret:
    name: test-secret  # an embedded LocalObjectRef
    items:
    - key: data-1
      path: special/path/__special_thing-1.dat
    - key: data-2
      path: special/path/__special_thing-2.dat